SymmetricProject/finsums.lean

```diff
@@ -1,21 +1,16 @@
 -- This is not part of the project, but was written in order for to practice manipulation of finite sums in Lean.
 
-import Mathlib.Data.Real.Basic
 import Mathlib.Data.Finset.Basic
-import Mathlib.RingTheory.MvPolynomial.Basic
-import Mathlib.RingTheory.MvPolynomial.Symmetric
 import Mathlib.Algebra.BigOperators.Basic
+import Mathlib.Tactic.Linarith
 
-open MvPolynomial
 open Finset
 open BigOperators
 
 example (n : ℕ) : ∑ t in range n, 2*t + n = n * n := by
   induction' n with n hn
-  simp
-  simp [Finset.sum_range_succ, hn]
-  rw [Nat.succ_eq_add_one]
-  have h : (∑ t in range n, 2 * t) + 2 * n + (n + 1) = (∑ t in range n, 2 * t) + n + n + (n + 1) := by ring 
-  rw [h, hn]
-  ring
-  
+  · simp
```

Comment:
The `· ` "focuses" on a goal, so only one of the two goals is visible to Lean. At the end of the `· ` block the goal must be discharged.

These are basic "structuring" tactics that make the proof scripts relatively more readable.